Develop

BenjaminGarza opened the pull request.

BenjaminGarza committed:
Initial commit

BenjaminGarza committed:
Rough navbar layout

BenjaminGarza committed:
Laying out content

BenjaminGarza committed:
Added Culture&Travel and Latest News sections

BenjaminGarza committed:
Nav-bar now is sticky,  added links, Expanded Featured Stories

BenjaminGarza committed:
Very rough layout

Added html for placing content, added rough footer

BenjaminGarza committed:
Merge pull request #1 from BenjaminGarza/Feature-Getting-started

Feature getting started

BenjaminGarza committed:
More adjustments to page layout, reworking containers for "other content"

BenjaminGarza committed:
Merge pull request #2 from BenjaminGarza/Layout-and-responsiveness

More adjustments to page layout, reworking containers for "other cont…

BenjaminGarza committed:
Updated pictures and text on "in the magazine"

BenjaminGarza committed:
Made header display properly, added "in-depth stories at the bottom

BenjaminGarza committed:
Added some unique card photos, worked on header and footer

Also worked on card titles

BenjaminGarza committed:
Updating articles

BenjaminGarza committed:
"Sign up for newsletter" updated

BenjaminGarza committed:
Content and styling

BenjaminGarza committed:
top nav bar, media query for second nav and hamburger icon

BenjaminGarza committed:
Responsive design changes

Finally fixed bug with subscribe and newsletter not displaying side by side.
Added responsiveness to in-the-mag and editors picks.
Added hamburger icon

BenjaminGarza committed:
Editors picks responsive changes

BenjaminGarza committed:
Worked on drop down menu

BenjaminGarza committed:
Validation fixes before merging to develop

BenjaminGarza committed:
Merge pull request #3 from BenjaminGarza/Styling-and-Responsiveness

Styling and responsiveness

BenjaminGarza committed:
Update to README

BenjaminGarza committed:
Merge pull request #4 from BenjaminGarza/Styling-and-Responsiveness

Update to README

Oghenebrume50 commented:
Hi @BenjaminGarza  This is great work, I have a few corrections for you to implement though.
 - [x] The NEWSWEEK logo at the top should be centred and there should be a date to the left of it.
- [x] around the 669px width breakpoint the more stories section looks completely off, could you fix that

Great  work so far, make these changes and submit for re-review. BTW I noticed you didn't use [Github flow](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/) please check it out but don't bother using it on this project, just keep it in mind for your next HTML projects, that's what you should be using.

BenjaminGarza committed:
Added date on left of nav, moved newsweek to middle, fixed break point display error

BenjaminGarza committed:
Fixed Html validation error

bolah2009 commented:
Currently under review

bolah2009 commented on Main.css:
```diff
@@ -0,0 +1,412 @@
+    }
+}
+@media only screen and (min-width: 768px){
+      
```
### `HINT`

Consider adding a smaller font size for the navigation bar for smaller screens. Also, the footer can use some repositioning for smaller screens. See illustration below:

| Current | Preferred |
| :-----: | :------: |
| ![bootstrap footer current](https://user-images.githubusercontent.com/36057474/62749163-47baff00-ba53-11e9-8352-3ff6abb578c0.png) | ![bootstrap footer preffered](https://user-images.githubusercontent.com/36057474/62749162-47226880-ba53-11e9-9015-a50c90016558.png) |

bolah2009 commented on index.html:
```diff
@@ -0,0 +1,952 @@
+                </div>
+                <div class="row">
+                    <div class="col">
+                <h6 class="more-stories-title">MORE STORIES</h6>
```
### `HINT`

Consider styling this and all other occurrences. (For example, giving it font color and weight as in the source page). See below:

| Current | Preferred |
| :-----: | :-----: |
| ![more stories current](https://user-images.githubusercontent.com/36057474/62750140-e5fc9400-ba56-11e9-961b-d508e5f79409.png) | ![more stories preffered](https://user-images.githubusercontent.com/36057474/62750149-edbc3880-ba56-11e9-9e0e-d3943c93ffa1.png) |

#### Do this for all other occurrences.

bolah2009 reviewed (changes requested):
## Almost there  :smile:  

The clone looks nice, however, I would suggest making sure **nothing is messed up** compared to the source page. There are some styling that is not implemented in the clone.
See below (check the full [page](https://www.newsweek.com/) for more section that needs to be worked on:

| Current | Preferred |
| :-----: | :-----: |
| ![bootstrap full current](https://user-images.githubusercontent.com/36057474/62749614-f3b11a00-ba54-11e9-9f7f-cb9db7bc0e02.gif) | ![bootstrap full preffered](https://user-images.githubusercontent.com/36057474/62749629-03306300-ba55-11e9-93bd-4ebc1555cc2b.gif) |

You can recreate this issue (Image above) by using the console on chrome **(Inspect `Ctrl+Shift+I`)** and the **Toggle device toolbar** `Ctrl+Shift+M`. This way you can simulate how the app will render on small to widescreen devices.


#### Friendly reminder: You should use Github Flow and [Linter](https://github.com/microverseinc/linters-config#linters-config) in this project. However, changing it now would take too much time. Please make sure you use Github Flow and [Linter](https://github.com/microverseinc/linters-config#linters-config) in the next project. You will find instructions in the Pathwright lesson description.


Good luck with your next submission.

Cheers
[Bola](https://bolabuari.com/)

BenjaminGarza committed:
Menu now resizes below 450px

BenjaminGarza committed:
Changed footer to display in a column under 768px. redid more stories styling

Torres-ssf commented:
### Pull Request Under Review By Sérgio (TSE)

Torres-ssf reviewed (changes requested):
### Hi there @BenjaminGarza 
You worked hard making these changes. But I notice that the stickler is not verifying your project. Maybe this is happening because it's not set up properly. Please check the instruction on the pathwrite on how to set up the linter and make it work on this project. You will also need to fix errors(if the linter found any). I know that in the past we didn't pay much attention to it but linting is important to reduce errors and improve the overall quality of your code. Using the linter also gives a professional look in your repository. 

Please kindly make this change and submit for code review again.
Happy coding :smile: 

[FOR FUTURE PROJECT]
 You should use Github/Git Flow in future projects. It's really easy to use it and will make your repository looks more professional. You will find instructions in the Patwright lesson description.

BenjaminGarza committed:
Create stylelint.config.js

BenjaminGarza committed:
Needed period in front of (.)stickler.yml

BenjaminGarza committed:
Fix css validation errors

BenjaminGarza committed:
Fixed unclosed block

BenjaminGarza committed:
Sinlge space after css class before {}

BenjaminGarza committed:
Attempting fixer option in stickler

BenjaminGarza committed:
Set fixers to enabled

stickler-ci committed:
Fixing style errors.

stickler-ci committed:
Fixing style errors.

BenjaminGarza committed:
Fixed 8 linter errors

BenjaminGarza committed:
Fixed two decending specifity errors

stickler-ci committed:
Fixing style errors.

BenjaminGarza committed:
Stickler config file name typo

Ispirett reviewed (changes requested):
Hey @BenjaminGarza @Oghenebrume50  Great! job with the changes requested by @Torres-ssf  and @bolah2009 
![image](https://user-images.githubusercontent.com/38801423/62908122-703d4480-bd44-11e9-9115-42025ab7f341.png). 

#### There are just a few things to address 
##### The Newsweek logo is too big
![fix1](https://user-images.githubusercontent.com/38801423/62991266-f2993780-be1c-11e9-8149-82f117382afe.png)

##### The content on the sides of the newsweek logo are to small.
![fix2](https://user-images.githubusercontent.com/38801423/62991271-fdec6300-be1c-11e9-8782-acdb5ddfd4a8.png)

stickler-ci committed:
Fixing style errors.

BenjaminGarza committed:
Media query ending in wrong place, cause of nav menu display bug

Also noticed scroll bar under "US" side content, changed pictures size to remove scroll bar

Torres-ssf commented:
### Pull Request Under Review By Sérgio (TSE)

Torres-ssf reviewed (approved):
### Hi there @BenjaminGarza 
I'm giving this project a :+1: 
Go on and merge to master. Good luck with the next one.

Friendly reminder: You should use Github/Git Flow in this project. However, changing it now would take too much time. Please make sure you use Github/Git Flow in the next project. You will find instructions in the Patwright lesson description.
